src/Controller/Api/NodeController.ts

```diff
@@ -8,17 +8,19 @@ import {ServiceError} from "../../Services/ServiceError";
 
 export class NodeController {
 
-    private nodeService: NodeService
+    private nodeService: NodeService;
 
     constructor(nodeService: NodeService) {
         this.nodeService = nodeService;
     }
 
     public async createNode(req: ValidatedRequest<CreateNodeRequestSchema>, res: Response): Promise<any> {
         try {
-            const {token, nodeInfo} = req.body;
+            const {token, nodeInfo, notifications} = req.body;
             const userId = res.locals.userId;
-            const result = await this.nodeService.createNode(nodeInfo.url, token, nodeInfo.address, userId);
+            const result = await this.nodeService.createNode(
```

Comment:
We don't support enabling notifications when creating node from daemon so we can assume they are enabled by default.